src/js/pensions/config/form.js

```diff
@@ -133,9 +138,48 @@ const formConfig = {
               combatSince911
             }
           }
+        },
+        reserveAndNationalGuard: {
+          path: 'military/reserveAndNationalGuard',
+          title: 'Reserve and National Guard',
+          uiSchema: {
+            'ui:title': 'Reserve and National Guard',
+            nationalGuardActivation: {
+              'ui:title': 'Are you currently on federal active duty in the National Guard?',
+              'ui:widget': 'yesNo'
+            },
+            nationalGuard: {
+              'ui:title': 'What is the information for your unit?',
+              'ui:options': {
+                expandUnder: 'nationalGuardActivation',
+              },
+              name: {
+                'ui:title': 'Name of Reserve/NG unit'
+              },
+              address: address.uiSchema(),
+              phone: phoneUI(),
+              date: dateUI
```

Comment:
`dateUI` is a function (you'll see a prop type error in the dev console for this page because of it).